src/components/screens/monitor/transactions/overview/overview.js

```diff
@@ -104,14 +104,14 @@ const normalizeNumberRange = (distributions) => {
     '0.01_0.1': '0 - 10 LSK',
     '0.1_1': '0 - 10 LSK',
     '1_10': '0 - 10 LSK',
-    '10_100': '10 - 100 LSK',
-    '100_1000': '100 - 1000 LSK',
-    '1000_10000': '1000K - 10,000 LSK',
-    '10000_100000': '10,000 - 100,000 LSK',
-    '100000_1000000': '100,000 - 1,000,000 LSK',
-    '1000000_10000000': '1,000,000 - 10,000,000 LSK',
-    '10000000_100000000': '10,000,000 - 100,000,000 LSK',
-    '100000000_1000000000': '100,000,000 - 1,000,000,000 LSK',
+    '10_100': '11 - 100 LSK',
+    '100_1000': '101 - 1000 LSK',
+    '1000_10000': '1001 - 10,000 LSK',
+    '10000_100000': '10,001 - 100,000 LSK',
```

Comment:
@ElnazShahidian Since we're adding commas to all the larger numbers, we might as well do so for `1000` and `1001`

Reply:
Good point

Reply:
I think it would be ugly and I haven't seen anyone do it for 1k

Reply:
@vibratious Fair enough